lib/order.rb

```diff
@@ -0,0 +1,84 @@
+require 'twilio-ruby'
+require './.env.rb'
+
+class Order 
+
+  attr_reader :total, :menu, :basket
```

Comment:
`total` should be an `attr_accessor` since you are writing to the instance variable.